web-server/app/views/searchBox.scala.html

```diff
@@ -103,6 +103,12 @@
                 }
             }
         }
+        document.getElementsByName("query")[0].addEventListener('keyup', function (event) {
```

Comment:
I think that event worth adding to other inputs also.

Reply:
Yes, it should work for all text inputs in the search form.

Reply:
got it - will fix